apps/console/src/features/identity-providers/components/forms/authenticators/sms-otp-authenticator-form.tsx

```diff
@@ -338,6 +374,43 @@ export const SMSOTPAuthenticatorForm: FunctionComponent<SMSOTPAuthenticatorFormP
         return errors;
     };
 
+    /**
+     * Handle enable/disable SMS OTP.
+     *
+     * @param event - Event.
+     * @param data - Data.
+     */
+    const handleUpdateSMSPublisher = (event, data) => {
+
+        if (data.checked) {
+            // Add SMS Publisher when enabling the feature.
+            addSMSPublisher().then((response: NotificationSenderSMSInterface) => {
+                setEnableSMSOTP(true);
+                setIsReadOnly(false);
+            }).catch((error: AxiosError) => {
+                dispatch(addAlert({
+                    description: error?.response.data.description ||
```

Comment:
Lets use `i18n` messages instead of showing server errors.

Reply:
resolved in https://github.com/wso2/identity-apps/pull/3699/commits/087b1e409184a950309763eb50cbf62430fd51ce